torchao/quantization/quant_api.py

```diff
@@ -650,6 +664,7 @@ def int4_weight_only(
          size is more fine grained, choices are [256, 128, 64, 32]
         `layout`: layout type for quantized tensor, default is `TensorCoreTiledLayout(inner_k_tiles=8)`
         `use_hqq`: whether to use hqq or default quantization mode, default is False
+        `zero_point_domain`: data type of zeros points, choices are [None(default), ZeroPointDomain.FLOAT, ZeroPointDomain.INT, ZeroPointDomain.NONE]
```

Comment:
nit: probably adding that `None` means we'll set zero_point_domain based on layout

Reply:
https://github.com/pytorch/ao/pull/1401/commits/e4237e006d11624633555ac40283ee6dd8349f34